tests/scripts/github_cc_reviewers.py

```diff
@@ -66,8 +67,36 @@ def find_reviewers(body: str) -> List[str]:
     if body is None:
         body = ""
 
-    to_add = find_reviewers(body)
-    print("Adding reviewers:", to_add)
+    new_reviewers = find_reviewers(body)
+    print("Found these reviewers:", new_reviewers)
+
+    if args.testing_reviews_json:
+        existing_reviews = json.loads(args.testing_reviews_json)
+    else:
+        github = GitHubRepo(token=os.environ["GITHUB_TOKEN"], user=user, repo=repo)
+        existing_reviews = github.get(f"pulls/{number}/reviews")
+
+    existing_review_users = [review["user"]["login"] for review in existing_reviews]
```

Comment:
should we also analyze the state of the review?

Reply:
since this is just for notifying I don't think we need to, regardless of whether its approved/pending/changes requested the bot shouldn't take any action if the person is already in the review box